server/index.js

```diff
@@ -141,36 +157,54 @@ function logRooms() {
 
 function showVotes(roomId) {
     const roomTickets = tickets.filter(p => p.roomId == roomId);
-
+    // find the text in the gametype where the index is the closest
+    let closest = 0;
+    const average = getAverage(roomId);
     if (roomTickets) {
-        const average = getAverage(roomId);
-        const fib = [0, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
-        let closest = 0;
+        const fib = gameType.find(p => p.roomId == roomId).gameType.values
+        closest = 0;
         let smallestDiff = Number.MAX_VALUE;
-        for (const number of fib) {
+        for (const item of fib) {
+            // get the index of the item in the fib array
+            const number = fib.indexOf(item);
             const difference = Math.abs(number - average);
-            if (difference < smallestDiff) {
+            if (difference <= smallestDiff) {
                 smallestDiff = difference;
                 closest = number;
             }
         }
-
         const ticket = roomTickets.find(f => f.votingOn);
         if (ticket) {
-            ticket.score = closest;
+            ticket.score =  gameType.find(p => p.roomId == roomId).gameType.values[closest];
         }
     }
 
-    io.to(roomId).emit('show');
+    const nearest = gameType.find(p => p.roomId == roomId).gameType.values[closest];
+    let avg = '';
+    if (average.toString().indexOf(".") > -1) { 
+        // get the full number rounding it down
+        let myAvg = Math.floor(average);
+        avg = gameType.find(p => p.roomId == roomId).gameType.values[myAvg]
+        // set myAvg to the full number rounding up
+        myAvg = Math.ceil(average);
+        avg += ' ' + gameType.find(p => p.roomId == roomId).gameType.values[myAvg];
+    }
+    else {
+        avg = gameType.find(p => p.roomId == roomId).gameType.values[average];
+    }
+    io.to(roomId).emit('show', { average: avg, closest: nearest });
 }
-
+ 
 function getAverage(roomId) {
```

Comment:
How come the average looks like this?
![image](https://github.com/LukeGarrigan/planfree.dev/assets/12545967/a4b3eee3-996d-4040-8930-71221a931f97)